.circleci/config.yml

```diff
@@ -58,22 +58,51 @@ commands:
             ./deps/readies/bin/getpy3
             ./deps/readies/bin/getpy2
 
+  save-tests-logs:
+    steps:
+      - run:
+          name: Cleanup test log dir
+          command: |
+            if [[ -d tests/flow/logs ]]; then
+              cd tests/flow/logs
+              rm -f *.aof *.rdb
+            fi
+            if [[ -d tests/tck/logs ]]; then
```

Comment:
Why not just remove both. rm -f won't complain:

Reply:
We want to keep `*.log` and remove `*.aof *.rdb`, which just take space as artifacts and slow down the artifact upload process in CI.
Do you mean like: `rm -f tests/tck/logs/*.aof tests/tck/logs/*.rdb`?

Reply:
Yeah or something like ```rm -f tests/tck/logs/*.{aof|rdb}```

Reply:
Nice. Will work with a comma: `rm -f tests/tck/logs/*.{aof,rdb}`